src/foam/u2/view/FileView.js

```diff
@@ -25,10 +25,11 @@ foam.CLASS({
         .add(this.slot(function(data) {
           return ! data ? this.E('span') : this.E('a')
             .attrs({
-              href: data.data && data.data.blob ?
+              href: (data.data && data.data.blob) || data.dataString ?
                 URL.createObjectURL(data.data.blob) :
```

Comment:
Couldn't data.data.blob or data.data be null on this line if data.dataString isn't?

Reply:
They can't be null if data.dataString isn't. I deleted the check